resources/openshift/templates/jboss64-image-streams.json

```diff
@@ -0,0 +1,152 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-image-streams",
+        "annotations": {
+            "description": "ImageStream definitions for JBoss Middleware products.",
+            "openshift.io/provider-display-name": "Red Hat, Inc."
+        }
+    },
+    "items": [
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-datagrid71-client-openshift",
```

Comment:
there should not be datagrid image streams defined in this file

Reply:
Since JDV install requires the JDG client and only a certain version, wouldn't it be better to control that here?

Reply:
This will conflict with whatever the datagrid product is releasing (i.e. which image stream gets installed, yours or theirs?).

Reply:
We can state the requirement in documentation and/or errata, but we should definitely not have it here IMHO